### DISCUSSION TITLE
Ubuntu上挂载alist的OneDrive速度慢

### DISCUSSION THREAD

**Discussion by AbyssSkb** [AlistGo/alist#3445] [General]
如题，用rclone通过webdav方式挂载alist中的OneDrive，速度很慢，只有200k，有大佬有解决方案吗？

**Comment by youtehub** [comment]
onedrive 服务器在国外，用 外网代理 试试吧，那种不限流量的 梯子，还有 Cloudflare Workers部署Onedrive直链程序并实现反代下载

**Reply by AbyssSkb** [reply]
可是直接用下载器下载alist中的OneDrive文件可以直接跑到满速，感觉是rclone的问题

**Reply by nyakochao** [reply]
> onedrive 服务器在国外，用 外网代理 试试吧，那种不限流量的 梯子，还有 Cloudflare Workers部署Onedrive直链程序并实现反代下载

有国外shadowrocket这种节点，但是怎么用外网代理呢，吃不吃我服务器的流量和带宽
